lib/cli/invocation/command/addBootstrapCommands.ts

```diff
@@ -145,11 +145,20 @@ function addSdmGenerator(yargs: YargBuilder) {
                 if (chm.tags.includes("spring" as any)) {
                     await doAfterSpringSdmCreation();
                 }
+                if ((hr as any).generatedRepositoryUrl) {
+                    infoMessage(`To run your new SDM:
+cd ${removeUrlScheme((hr as any).generatedRepositoryUrl)};
+atomist start`);
```

Comment:
Shouldn't that mention `atomist start --local` because they are unlikely to have a local config configured?

Reply:
Our Getting Started instruction now want people to connect to the team. Local SDM mode is no longer in the flow we're leading them down.